package.json

```diff
@@ -45,6 +45,7 @@
     "docusaurus-plugin-openapi-docs": "^4.3.7",
     "docusaurus-theme-openapi-docs": "^4.3.7",
     "mixpanel-browser": "^2.64.0",
+    "npx": "^10.2.2",
```

Comment:
I don't think adding the package should be part of this PR 🤔

Reply:
Good point. That's somehow slipped my publish commit